ouroboros-consensus/src/Ouroboros/Consensus/Mempool/API.hs

```diff
@@ -192,14 +199,14 @@ data Mempool m blk idx = Mempool {
 -- when the transaction has been removed from the mempool between snapshots.
 --
 data MempoolSnapshot blk idx = MempoolSnapshot {
-    -- | Get all transactions in the mempool snapshot along with their
-    -- associated ticket numbers (oldest to newest).
-    getTxs      :: [(GenTxId blk, GenTx blk, idx)]
-
-    -- | Get all transactions in the mempool snapshot, along with their
-    -- associated ticket numbers, which are associated with a ticket number
-    -- greater than the one provided.
-  , getTxsAfter :: idx -> [(GenTxId blk, GenTx blk, idx)]
+    -- | Get all transactions (oldest to newest) in the mempool snapshot along
+    -- with their ticket number and size.
+    getTxs      :: [(GenTx blk, idx, TxSizeInBytes)]
+
+    -- | Get all transactions (oldest to newest) in the mempool snapshot,
+    -- along with their ticket number and size, which are associated with a
+    -- ticket number greater than the one provided.
+  , getTxsAfter :: idx -> [(GenTx blk, idx, TxSizeInBytes)]
```

Comment:
I don't think we need to return the sizes here. The caller can do that if they need it can't they? It's cheap to do.

Reply:
Same for `getTxs`, I suppose.